pkg/graph/resolvers/system_intake.go

```diff
@@ -41,13 +42,21 @@ func CreateSystemIntakeContact(
 	ctx context.Context,
 	store *storage.Store,
 	input models.CreateSystemIntakeContactInput,
+	getAccountInformation userhelpers.GetAccountInfoFunc,
 ) (*models.CreateSystemIntakeContactPayload, error) {
 	contact := &models.SystemIntakeContact{
 		SystemIntakeID: input.SystemIntakeID,
 		EUAUserID:      input.EuaUserID,
 		Component:      input.Component,
 		Role:           input.Role,
 	}
+	contractsUserAccount, err := userhelpers.GetOrCreateUserAccount(ctx, store, store, input.EuaUserID, false, getAccountInformation)
```

Comment:
s/contracts/contact?
```suggestion
	contactUserAccount, err := userhelpers.GetOrCreateUserAccount(ctx, store, store, input.EuaUserID, false, getAccountInformation)
```